.github/workflows/ci.yaml

```diff
@@ -26,12 +30,16 @@ jobs:
         run: black --check clinvar_ingest test
       - name: Sort imports
         run: isort --check-only clinvar_ingest test
+      # Out of curiousity, what we seeing pylint catch that ruff doesn't?
+      # We might consider turing on every possible ruff option just to see what happens,
+      # especially since this is greenfield repo.
       - name: Ruff lint
         run: ruff check clinvar_ingest test
       # Pylint catches some things that Ruff doesn't yet support
       # Run error-only mode `--disable=C,R,W` == `-E`
       - name: Pylint
         run: pylint --disable=C,R,W clinvar_ingest
+      # We should try to get mypy/pyright working here since this is a greenfield project.
```

Comment:
+1 to more type checking

I haven't used mypy before but I have seen people recommend it before and I've heard good things